include/swift/Option/Options.td

```diff
@@ -485,6 +485,12 @@ def static_stdlib: Flag<["-"], "static-stdlib">,
 def no_static_stdlib: Flag<["-"], "no-static-stdlib">,
   Flags<[HelpHidden,DoesNotAffectIncrementalBuild]>,
   HelpText<"Don't statically link the Swift standard library">;
+def toolchain_stdlib_rpath: Flag<["-"], "toolchain-stdlib-rpath">,
+  Flags<[HelpHidden,DoesNotAffectIncrementalBuild]>,
+  HelpText<"Add an rpath entry for the toolchain's standard library, rather than the OS's">;
+def no_stdlib_rpath: Flag<["-"], "no-stdlib-rpath">,
+  Flags<[HelpHidden,DoesNotAffectIncrementalBuild]>,
+  HelpText<"Don't add any rpath entries.">;
```

Comment:
Is there any reason we might want an `-os-stdlib-rpath` option to force the default `-rpath /usr/lib/swift` behavior?

Reply:
:-/ Eh. I can't think of one, and you can always recreate it with `-no-stdlib-rpath -Xlinker -rpath -Xlinker /usr/lib/swift`.